intermine_boot/intermine_docker.py

```diff
@@ -87,6 +86,15 @@ def up(options, env):
 
     _create_volumes(env)
 
+    # NOTE: This assumes the path provided points to a folder named biotestmine.
```

Comment:
Maybe it is better if we can extract the folder name from the provided path itself?
This will be a much more predictable behaviour and one less thing for a user to remeber.

Reply:
done.